pennylane/labs/resource_estimation/ops/op_math/controlled_ops.py

```diff
@@ -490,6 +988,16 @@ def _resource_decomp(
 
     @property
     def resource_params(self) -> dict:
+        r"""Returns a dictionary containing the minimal information needed to compute the resources.
+
+        Resource parameters:
+            num_ctrl_wires (int): the number of qubits the operation is controlled on
+            num_ctrl_values (int): the number of control qubits, that are controlled when off
```

Comment:
"the number of control qubits, that are controlled when off" Not sure what this means.

Reply:
Fixed. It means to control the qubit when its in state |0>